app/views/shared/_header.html.erb

```diff
@@ -26,7 +26,7 @@
           <li class="nav-item sub-menu-toggler"><a href="#" class="nav-link text text-menu -light">Participate</a>
             <ul class="sub-menu">
               <li><%= link_to 'Campaigns', campaigns_path, {class: 'text text-menu -dark'} %></li>
-              <li><%= link_to 'Mapathons', "/", {class: 'text text-menu -dark'} %></li>
+              <li><%= link_to 'Mapathons', mapathon_path, {class: 'text text-menu -dark'} %></li>
```

Comment:
similar to the previous comment, the path name should be events_path (with event in the plural)